### PR TITLE
dev 1700 black indexd

### PR DESCRIPTION
**Pull request by qiaouchicago**
black index 
no logic change

### CODE REVIEW

**Review by ProfOak — APPROVED** [review]
:tada:

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -19,3 +14,13 @@ repos:
       - id: end-of-file-fixer
       - id: no-commit-to-branch
         args: [--branch, develop, --branch, master, --pattern, release/.*]
+  - repo: https://github.com/psf/black
+    rev: 19.10b0
+    hooks:
+      - id: black
+        additional_dependencies: [click==8.0.4]
+  - repo: git@github.com:Yelp/detect-secrets
```
Comment (stilesj-uchicago) [inline .pre-commit-config.yaml:22]: Good job moving detect-secrets to the end.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -19,3 +14,13 @@ repos:
       - id: end-of-file-fixer
       - id: no-commit-to-branch
         args: [--branch, develop, --branch, master, --pattern, release/.*]
+  - repo: https://github.com/psf/black
+    rev: 19.10b0
```
Comment (stilesj-uchicago) [inline .pre-commit-config.yaml:18]: I think you can use black 22.3.0 if you're using an isolated python 3.7 to run pre-commit. https://github.com/NCI-GDC/cookiecutter-pystones/blob/8727386f7d9b5554b8f7d2a6fd7e10a640465943/%7B%7B%20cookiecutter.package_name%20%7D%7D/.pre-commit-config.yaml#L27

**Review by stilesj-uchicago — APPROVED** [review]

**Review by kulgan — APPROVED** [review]

File: .pre-commit-config.yaml (hunk 2 above)
Comment (qiaouchicago) [reply .pre-commit-config.yaml:18]: I think 19.10b0 is the version that supports our current Jenkins pipeline. 
I will upgrade black and remove Jenkins.

**Pull request merged**